docs/customApps.md

```diff
@@ -74,6 +74,12 @@ Before running analysis, make sure that you have `kantra` installed
 1. `./get_latest_kantra_cli.sh` # fetches 'kantra' our analyzer tool and stores it in ../bin
 1. `cd ..`
 
+
+To ensure the analysis uses the necessary custom rules, follow these steps:
+
+1. Custom Rules: Download or create your custom rules and place them in the samples/custom_rules directory.
+2. Ignore Default Rules: The analysis script is set to ignore default Kantra rules. You can review or adjust this in example/analyze.sh if needed.
```

Comment:
Calling this out make sense, let's expand to include the 2 places that we perform analysis so folks are aware.

Note that we have 2 different places that perform analysis and currently only one of them is set to ignore the default Kantra rules.

I believe the intent of this document has focused on the analysis of sample data

1) Analysis of the 'sample data', this is analysis that runs on several sample applications where have initial/solved states.  This analysis is done to simulate what integration will look like when we run with a Konveyor instance that an organization has populated with data.   For this specific use-case we are running the below to analyze the sample apps:

https://github.com/konveyor/kai/blob/main/samples/analyze_apps.py#L83
`cmd = f'time ./bin/kantra analyze -i {source_dir} -t "quarkus" -t "jakarta-ee" -t "jakarta-ee8+" -t "jakarta-ee9+" -t "cloud-readiness" --rules {os.path.dirname(__file__)}/custom_rules -o {full_output_dir} --overwrite'`

In the above, we are allowing Kantra to use the default rules and we supplement with the custom rules.

2) We also have another place we run analysis, client side
This client side usage could be from the example/run_demo.py or from running in IDE.
Currently we are ignoring the default rules for the client side with example/run_demo.py and we experimented with a few small tweaks.

https://github.com/konveyor/kai/blob/main/example/analyze.sh#L50
`time "${KANTRA_BIN}" analyze -i "${PWD}"/"${SOURCE_DIR}" "${SOURCE_ONLY}" --enable-default-rulesets=false -t "quarkus" -t "jakarta-ee" -t "jakarta-ee8" -t "jakarta-ee9" -t "cloud-readiness" --rules "${DEFAULT_RULES_DIR}" --rules "${CUSTOM_RULES_DIR}" -o "${OUTDIR}" --overwrite`

We use the `--enable-default-rulesets=false` option client side and then pass in the below directory to use:
https://github.com/konveyor/kai/tree/main/example/default_rules